cluster.go

```diff
@@ -5,6 +5,24 @@ import (
 	"time"
 )
 
+type ClusterConfigurationInput struct {
+	EnableCsiDriver    bool `json:"csiDriver"`
```

Comment:
This one is deprecated, the CSI driver is always enabled.

Reply:
btw the current Terraform provider has a bug (the json fields are reversed) this will also fix that.